### DISCUSSION TITLE
El Buyer Persona del proyecto 1 sería el demandante o el abogado?

### DISCUSSION THREAD

**Discussion by DanielOe21** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#131] [Q&A] [answered]
Hola compañeros!!

Precisamente me encuentro haciendo esta parte de la práctica, y me generó una duda interesante:

El buyer persona es el usuario al que va dirigida la app web, para orientar el marketing y el diseño en torno a ello, normalmente se entiende que el cliente y el usuario son lo mismo, aunque en casos como este pues el cliente es el despacho de abogados y el usuario final sería la persona interesada en hacer la demanda. De hecho, este tema se mencionó en el live del jueves pasado en conjunto con el invitado colombiano.

¿Qué piensan ustedes? Para mi, el buyer persona es el usuario que va a demandar.

**Comment by GloriaCoralCerecedo** [comment]
Sería el demandante 👀

**Reply by DanielOe21** [reply]
¿Aunque técnicamente yo le esté vendiendo la app al despacho de abogados?

**Reply by JonathanJRodriguez** [reply]
@DanielOe-21 Hola Daniel, realmente la venta al despacho de abogados ya estaría hecha para este punto, de lo contrario no se invertirían los recursos para realizar una propuesta tan detallada. Ahora formarían un equipo de trabajo el despacho de abogados y la empresa desarrolladora de software. El usuario final se convertiría en el cliente de la aplicación. 
Al menos eso pienso.
Saludos.

**Reply by DanielOe21** [reply]
Concuerdo con ello @JonathanJRodriguez , pero me pregunto ¿Qué pasaría si en realidad esta fuera una solicitud del despacho de abogados y hubiera otros competidores? o sea, que no estuviera cerrado el trato y esto fuera la presentación de una maqueta para ganar el contrato

**Comment by CharlieC57** [comment, marked as answer]
Realmente puedes hacer un buyer persona para ambos. Aunque el usuario principal es el demandante, para el administrador este sistema le ayuda a condensar la información de los casos y llevar el control de los mismos así como de los pagos. Si tienes duda de acerca de quienes son tus buyer persona solo preguntate quiénes son las personas reales que usan el sistema.

¡Saludos!
